instrumentation/jdbc/src/main/java/io/opentelemetry/javaagent/instrumentation/jdbc/JDBCConnectionUrlParser.java

```diff
@@ -68,7 +70,7 @@
    */
   JTDS_URL_LIKE() {
     @Override
-    DBInfo.Builder doParse(String jdbcUrl, DBInfo.Builder builder) {
+    Builder doParse(String jdbcUrl, Builder builder) {
```

Comment:
Is the rename to generic name Builder intended?

Reply:
It's not rename, it's re-import! No, it was not intended.

Reply:
Reverted

Reply:
@jkwatson I just talked about this issue at the last meeting and we run into it in the same week :P It does make me consider biting the bullet and replacing AutoValue with top-level builders if it can prevent this sort of trouble for end users.

Reply:
I like `DBInfo.Builder` better than `DBInfoBuilder`, but ya, I see what you're talking about now, and would not disagree with such a change